Scripts/accountDataVerification.js

```diff
@@ -0,0 +1,25 @@
+//메일 검증용 API링크 https://bootcamp-api.codeit.kr/docs
+
+const emailRegEx = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+const pwRegEx = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
+
+
+const EmailRegisterStatusInServer = async function (inputtedAccountEmail) {
+   const EmailWhichWantToBeCheck = {email : inputtedAccountEmail}
```

Comment:
혹시 참고하신 naming convention이 있으신걸까용..?
무난한 이름으로 추천해봅니다..

```suggestion
   const requestBody = { email : inputtedAccountEmail }
```

Reply:
위에서 말씀드렸던 규칙에 따라 작성해보려고 노력한 흔적입니다. 
네이밍을 늘려 명확하게 구분이 가도록 해보려 한건데 이건 좀 실패였나 보네요;;